docs/library/network.rst

```diff
@@ -276,18 +276,30 @@ For example::
         argument is passed. Otherwise, query current state if no argument is
         provided. Most other methods require active interface.
 
-    .. method:: wlan.connect(ssid, password)
+    .. method:: wlan.connect(ssid, password, [bssid])
 
         Connect to the specified wireless network, using the specified password.
 
+        The `bssid` argument, if specified, restricts connection to an AP with
+        given hardware address.
+        The format of this argument should match the `bssid` field returned from
+        the scan() method (binary form). 
+
+        Usually the `bssid` argument is not needed.
+        Useful scenarios are:
+            * Connecting to a hidden network
+            * Connecting to specific AP that shares the same SSID with other APs
```

Comment:
I'd say this paragraph is not needed.  If people need to use bssid they probably know why they need to use it.